appendices/VK_EXT_map_memory_placed.adoc

```diff
@@ -0,0 +1,60 @@
+// Copyright 2017-2022 The Khronos Group Inc.
+//
+// SPDX-License-Identifier: CC-BY-4.0
+
+include::{generated}/meta/{refprefix}VK_EXT_map_memory_placed.adoc[]
+
+=== Other Extension Metadata
+
+*Last Modified Date*::
+    2022-11-22
+*IP Status*::
+    No known IP claims.
+*Interactions and External Dependencies*::
+  - Interacts with apitext:VK_EXT_external_memory_host
+*Contributors*::
+  - Jason Ekstrand, Collabora
+  - Tobias Hector, AMD
+  - James Jones, NVIDIA
+  - Georg Lehmann, Valve
+  - Derek Lesho, Codeweavers
+
+=== Description
+
+This extension allows a client to request that flink:vkMapMemory2EXT
+attempt to place the memory map at a particular virtual address.
+
+include::{generated}/interfaces/VK_EXT_map_memory_placed.adoc[]
+
+=== Issues
+
+1) What should happen if the implementation cannot place the memory map at
+the requested address?
+
+*RESOLVED*: The memory object should be left unmapped and
+flink:vkMapMemory2EXT should return ename:VK_ERROR_MEMORY_MAP_FAILED.
+
+2) How can a client atomically re-reserve the address range on unmap?
+
+*RESOLVED*: When `VK_MEMORY_UNMAP_RESERVE_BIT_EXT` is passed to
+`vkUnmapMemory2EXT()`, the implementation unmaps the memory range in such a
+way that the range is automatically re-reserved.  With `mmap()`, this is
+accomplished by simply mapping over the range with another anonymous
+mapping.  However, allowing this with `vkMapMemory()` would break Vulkan's
+concept of when a memory object is or is not mapped.
+
+3) Should a placed memory map replace existing maps in the specified
+range or fail if a map exists?
```

Comment:
:woman_shrugging: IDK. I put most of the issues in both but I can drop them from the proposal.

Reply:
We've generally been dropping them from the appendix rather than the proposal - reason being the questions usually come up when you read the proposal, not the appendix. Suggest dropping them here? Not going to gate merging on this but it is weird to have them duped...